.travis.yml

```diff
@@ -48,7 +50,7 @@ jobs:
         # checkout master and ensure we are up2date
         - git checkout master
         - git pull || travis_terminate 1
-        - export TARGET_BRANCH=patch/1/update_go_utils
+        - export TARGET_BRANCH=patch/go_utils_${GO_UTILS_TARGET}
```

Comment:
I made the branch name dynamic now, such that it is clear that we are updating go-utils to a certain version/commit/... and such that the PRs don't get overwritten